format/Message.fbs

```diff
@@ -64,8 +64,11 @@ table Date {
 table Time {
 }
 
+enum TimeUnit: short { MILLISECOND, MICROSECOND }
```

Comment:
Also, Unix timestamps are seconds since the epoch. Not sure how commonly occurring these are in practice